mne/io/nicolet/nicolet.py

```diff
@@ -107,6 +107,15 @@ def _get_nicolet_info(fname, eog, ecg, emg, misc):
                  'ch_names': ch_names, 'description': None,
                  'buffer_size_sec': 10.})
 
+    if ch_type == 'eeg':
+        ch_coil = FIFF.FIFFV_COIL_EEG
+        ch_kind = FIFF.FIFFV_EEG_CH
+    elif ch_type == 'seeg':
+        ch_coil = FIFF.FIFFV_COIL_EEG
+        ch_kind = FIFF.FIFFV_SEEG_CH
+    else:
+        raise TypeError("Channel type not recognized. Available types are "
+                        "'eeg' and 'seeg'.")
     cal = header_info['conversion_factor'] * 1e-6
     for idx, ch_name in enumerate(ch_names):
         if ch_name in eog or idx in eog:
```

Comment:
something is weird with the logic here. Why do you need to have the `or` ? If `eog` is `auto`, then `eog` becomes a list of indices. But if the user provides a list of channels, then `eog` is a list of channels. Why don't you make `eog` a list of channels in both cases? They are mutually exclusive. You can't have `auto` as well as the user providing a list of channels. So, why the `or` condition? The semantics of `eog` changes depending on the conditions and it makes the code hard to read and debug.